templates/repo/empty.tmpl

```diff
@@ -50,14 +50,17 @@ git push -u origin {{.Repository.DefaultBranch}}</code></pre>
 git push -u origin {{.Repository.DefaultBranch}}</code></pre>
 								</div>
 							</div>
-							<script defer>
-								/* eslint-disable no-undef */
-								const cloneUrls = document.getElementsByClassName('clone-url');
-								if (cloneUrls) {
-									for (let i = 0; i < cloneUrls.length; i++) {
-										cloneUrls[i].textContent = (isSSH ? sshButton : httpsButton).getAttribute('data-link');
+							<script>
```

Comment:
Not that I understood the defer in the first place - but why don't we need it anymore?

Reply:
From MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer

Warning: This attribute (defer) **must not** be used if the src attribute is absent (i.e. for inline scripts), in this case it would have no effect.

Reply:
It was meant to execute after DOM parsing (e.g. `DOMContentLoaded`), but it seems inline scripts unfortunately don't support this flag, and I missed that fact.